src/Core/Context.idr

```diff
@@ -1659,7 +1664,7 @@ getSizeChange loc n
 
 export
 setVisibility : {auto c : Ref Ctxt Defs} ->
-                FC -> Name -> Visibility -> Core ()
+                FC -> Name -> WithDefault Visibility Private -> Core ()
```

Comment:
I think this should still be ` Visibility`  as you are not going to be explicitly setting `Default`, are you?